Dockerfile

```diff
@@ -21,6 +21,8 @@ RUN addgroup -S ${XP_USER_GROUP} \
 
 COPY --chown=${XP_USER}:${XP_USER_GROUP} management-service/bin/* /app/
 COPY --chown=${XP_USER}:${XP_USER_GROUP} management-service/database /app/database/
+# read+write access for owner/group but no write access for others
+RUN chmod -R 775 /app/xp-management
```

Comment:
* Not sure if the chmod has some unintended impact. Are we still able to build and deploy the internal image on top of this? Can it be tested prior to merging this?
* Do we need a similar change for the treatment service too?